ssl/ssl_key_share.cc

```diff
@@ -677,14 +684,20 @@ CONSTEXPR_ARRAY NamedGroup kNamedGroups[] = {
     {NID_X25519, SSL_GROUP_X25519, "X25519", "x25519"},
     {NID_SecP256r1Kyber768Draft00, SSL_GROUP_SECP256R1_KYBER768_DRAFT00, "SecP256r1Kyber768Draft00", ""},
     {NID_X25519Kyber768Draft00, SSL_GROUP_X25519_KYBER768_DRAFT00, "X25519Kyber768Draft00", ""},
+    {NID_SecP256r1MLKEM768, SSL_GROUP_SECP256R1_MLKEM768, "SecP256r1MLKEM768", ""},
+    {NID_X25519MLKEM768, SSL_GROUP_X25519_MLKEM768, "X25519MLKEM768", ""},
 };
 
 CONSTEXPR_ARRAY uint16_t kPQGroups[] = {
     SSL_GROUP_KYBER512_R3,
     SSL_GROUP_KYBER768_R3,
     SSL_GROUP_KYBER1024_R3,
+    SSL_GROUP_MLKEM768,
+    SSL_GROUP_MLKEM1024,
```

Comment:
Not actually used anywhere. Indifferent to whether to keep or not.

Reply:
I have a slight preference to keep `SSL_GROUP_MLKEM1024` IANA code point, but I'm not against removing it. If we're already pulling in the `SSL_GROUP_MLKEM768` code point from [draft-connolly-tls-mlkem-key-agreement](https://datatracker.ietf.org/doc/html/draft-connolly-tls-mlkem-key-agreement.html) I think we should also pull in the remaining code points in that draft for consistency; and if that code point is defined it should be marked as being a PQ code point.